packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -1082,39 +1084,40 @@ const AccountSwitcher = props => {
-                                            onClick={() => {
-                                                if (props.real_account_creation_unlock_date) {
-                                                    closeAccountsDialog();
-                                                    props.setShouldShowCooldownModal(true);
-                                                } else if (
-                                                    (can_manage_account_multi || can_manage_account_virtual) &&
-                                                    props.has_any_real_account
-                                                )
-                                                    props.openRealAccountSignup('manage');
-                                                else props.openRealAccountSignup(account);
-                                            }}
-                                            className='acc-switcher__new-account-btn'
-                                            secondary
-                                            small
-                                        >
-                                            {localize('Add')}
-                                        </Button>
-                                    </div>
-                                ))}
+                            {!has_cr_account &&
+                                getRemainingRealAccounts()
+                                    .filter(account => account === 'svg')
+                                    .map((account, index) => (
+                                        <div key={index} className='acc-switcher__new-account'>
+                                            <Icon icon='IcDeriv' size={24} />
+                                            <Text size='xs' color='general' className='acc-switcher__new-account-text'>
+                                                {getAccountTitle(
+                                                    account,
+                                                    { account_residence: props.client_residence },
+                                                    props.country_standpoint
+                                                )}
+                                            </Text>
+                                            <Button
+                                                id='dt_core_account-switcher_add-new-account'
+                                                onClick={() => {
+                                                    if (props.real_account_creation_unlock_date) {
+                                                        closeAccountsDialog();
+                                                        props.setShouldShowCooldownModal(true);
+                                                    } else if (
+                                                        (can_manage_account_multi || can_manage_account_virtual) &&
+                                                        has_cr_account
```

Comment:
Do we need this condition `has_cr_account` here?  This code block  from line 1087 will be executed when` !has_cr_account` is true . right?

Reply:
@thisyahlen-deriv The code block from line 1087 will be executed when `has_cr_account` is false. So inside the code block it will always be false and the else if condition  
```
 (can_manage_account_multi || can_manage_account_virtual) && has_cr_account
```
will not pass . isnt?

Reply:
Only if i add `has_cr_account ` it doesnt show the manage account modal. The behaviour is correct 🤔

Reply:
@thisyahlen-deriv i didnt get this. 😒
 In which scenario will this else part pass and show manage modal

Reply:
When I only have MF and try to add an Options account

Reply:
@thisyahlen-deriv the thing here is that you are checking `has_cr_account` to be false in the line 1087, then in the line 1108 you are checking `has_cr_account` to be true which is impossible

Reply:
Yes @mahdiyeh-deriv just discussed with @amina-deriv and remove the else if condition. 😸